Snapyr/Classes/SnapyrSDK.m

```diff
@@ -103,6 +118,36 @@ + (void)handleNoticationExtensionRequestWithWriteKey:(NSString *)writeKey bestAt
         contentHandler(bestAttemptContent);
         return;
     }
+    NSString *urlPath = snapyrData[@"imageUrl"];
```

Comment:
@igor-snapyr looking good. One more thing we need to figure out: `contentHandler` can really only be called once. (once it has been called, the notification completes, and any additional calls would be ignored)

We have potentially 2 different async things happening here before completion (category refetch and image fetch). Can we set this up to wait for both before completing? (if either one fails, the other should still be allowed to complete; if both fails, then complete with the original content)

Reply:
Sure, accounted for both